contrib/NeonVM/cell.c

```diff
@@ -67,7 +67,7 @@ void cell_copyCell(Cell *dest, const Cell *source)
     assert(dest != NULL);
 
     dest->number = source->number;
-    dest->string = _strdup(source->string);
+    dest->string = strdup(source->string);

```

Comment:
This will cause a problem if source->string is NULL (like if the source isn't a String cell at all).